app/routes/classifications_id.py

```diff
@@ -11,7 +10,7 @@
 def classifications_id(job_id):
     """Returns the status and the result of the job identified
     by the id specified in the path."""
-    redis_url = Configuration.REDIS_URL
+    redis_url = config.REDIS_URL
```

Comment:
why was this changed?

Reply:
I do not know. But thanks for the hint. I just changed it back since it makes no difference. Tested the functionalities and everything still works. Will also be reverted with next pull request